ignite/pkg/relayer/relayer.go

```diff
@@ -130,17 +128,16 @@ func (r Relayer) call(ctx context.Context, conf relayerconf.Config, path relayer
 		return relayerconf.Path{}, err
 	}
 
-	var reply relayerconf.Path
-
-	err = tsrelayer.Call(ctx, action, []interface{}{
+	args := []interface{}{
 		path,
 		srcChain,
 		dstChain,
 		srcKey,
 		dstKey,
-	}, &reply)
-
-	return reply, err
+		srcChain.ClientID,
```

Comment:
We already send srcChain and dstChain objets as a whole in the prev arguments. They should be available in the ts relayer wrapper. Can we please use this from there instead and remove `srcChain.ClientID`, `dstChain.ClientID`?